docs/_includes/ds6/button.html

```diff
@@ -210,12 +210,25 @@ <h3 id="button-expand">Expand Button</h3>
                 </span>
             </button>
         </div>
+        <div class="demo__inner">
+            <button aria-expanded="false" class="expand-btn expand-btn-example" aria-label="Expand/Collapse">
+                <span class="expand-btn__cell">
+                    <span></span>
+                    <span class="expand-btn__icon"></span>
+                </span>
+            </button>
+        </div>
         {% highlight html %}
 <button aria-expanded="false" class="expand-btn">
     <span class="expand-btn__cell">
         <span>Button</span>
         <span class="expand-btn__icon"></span>
     </span>
+</button>
+<button aria-expanded="false" class="expand-btn" aria-label="Expand/Collapse">
+    <span class="expand-btn__cell">
```

Comment:
Quick question though: I wonder whether we even need the cell container when we just have an icon. If I remember correctly, the cell is there to align text with an icon, so possibly not needed for an icon only case. I could be wrong though. Potentially we should be able to put any icon into a button and it look okay.